src/applications/vaos/containers/TypeOfAudiologyCarePage.jsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import SchemaForm from 'platform/forms-system/src/js/components/SchemaForm';
+import FormButtons from '../components/FormButtons';
+import {
+  openFormPage,
+  updateFormData,
+  routeToNextAppointmentPage,
+  routeToPreviousAppointmentPage,
+} from '../actions/newAppointment.js';
+import { getFormPageInfo } from '../utils/selectors';
+
+const initialSchema = {
+  type: 'object',
+  required: ['ccId'],
+  properties: {
+    ccId: {
```

Comment:
Might make more sense to call this `audiologyType`? Or if we think we might use it for other things, `secondaryTypeOfCareId`?